api/src/controllers/team.js

```diff
@@ -257,7 +259,7 @@ function updateTeam(req, res, next) {
     new: true,
     runValidators: true,
   })
-    .then((updatedTeam) => res.status(200).json(updatedTeam))
+    .then(() => res.status(200).json('Team has been updated'))
```

Comment:
Why send a success message when we can just send back the `updatedTeam` object? This appears to break REST to me.

Reply:
I did some research and it appears to be a contention topic: https://stackoverflow.com/a/798174/15507541
I'd still prefer returning the object over a success message though. A response code of `200` is perfectly enough to know that the team update operation was successful.

Reply:
@vgoh1 @kdav108 @ssha0054

Reply:
Thanks for pointing out the issue. Yeah, I was struggling whether which way we should adopt too. 
I decided to send back a message instead of an object eventually cause I thought it would be easier to let all the success/ error messages be handled in the backend and I didn't find a strong reason (other than would be helpful for testing in postman) to send back the object as it's just the same object sent from the frontend(the updateTeam endpoint) or the frontend not require the object to be sent back (the createTeam endpoint).

Reply:
To get technical since it is a patch operation, [convention](https://stackoverflow.com/questions/54666244/what-should-response-of-rest-full-api-for-patch-http-method) says returning `204 No Content` is possible if the client knows what the updated resource is going to be (which iirc they will cause this is called in the team profile page on the frontend). But to address the question of whether it should be returning the object or not, then yes it should to follow convention or it can return 204. On a related note: I agree with Akshay's comment [below](https://github.com/Researchify/Researchify/pull/140#discussion_r681403135) for reasons that I will post in that thread...

Reply:
Yes, I think one of the strong arguments to revert back to returning an object is discussed in the comment thread below. Basically:
- I think we don't need to return a success **message**, because the response code is sufficient
- As to Mable's point about finding a strong reason to send back the same object in update operations: I don't have a strong reason either, other than the fact that it makes the API cleaner, see this comment: https://github.com/Researchify/Researchify/pull/140#discussion_r681402217